Readability.js

```diff
@@ -112,7 +112,7 @@ Readability.prototype = {
   // All of the regular expressions in use within readability.
   // Defined up here so we don't instantiate them repeatedly in loops.
   REGEXPS: {
-    unlikelyCandidates: /banner|breadcrumbs|combx|comment|community|cover-wrap|disqus|extra|foot|header|legends|menu|modal|related|remark|replies|rss|shoutbox|sidebar|skyscraper|social|sponsor|ad-break|agegate|pagination|pager|popup|yom-remote/i,
+    unlikelyCandidates: /banner|breadcrumbs|combx|comment|community|cover-wrap|disqus|extra|foot|header|legends|menu|modal|related|remark|replies|rss|shoutbox|sidebar|skyscraper|social|sponsor|supplemental|ad-break|agegate|pagination|pager|popup|yom-remote/i,
```

Comment:
Add `supplemental` to remove `<div id="supplemental-1">` and `<div id="supplemental-2">` nodes from below dom structure. Then the `while (parentOfTopCandidate.tagName != "BODY" && parentOfTopCandidate.children.length == 1)` [loop][loop] will work to fix the nytimes issue.

```html
<article id="story" class="story theme-main   ">

    <div id="TragedyAd" class="ad tragedy-ad nocontent robots-nocontent"></div>

    <div class="dealbook-branding"></div>

    <header id="story-header" class="story-header"></header>

    <div class="story-body-supplemental">
        <div class="story-body story-body-1">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
        </div>
        <div class="supplemental first short" id="supplemental-1" data-between-flex-ads="true" data-pre-height="673" data-max-items="1" data-remaining="673" data-minimum="400" data-last-item-height="873" data-flex-ad-adjacency="true" data-post-height="673"></div>
    </div>

    <div class="story-interrupter" id="story-continues-1"></div>

    <div class="story-body-supplemental">
        <div class="story-body story-body-2">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
        </div>
        <div class="supplemental " id="supplemental-2" data-pre-height="2132" data-max-items="2" data-remaining="242" data-minimum="400" data-last-item-height="945" data-flex-ad-adjacency="false" data-post-height="2132"></div>
    </div>

    <div class="reader-satisfaction-survey prompt feedback-prompt story-content hidden"></div>

    <div id="storage-drawer" class="hidden"></div>
</article>
```
[loop]: https://github.com/mozilla/readability/pull/313/commits/8ec087f0ef72585ae03a63249868eb3b2689a9d8